### PR TITLE
Allow override of log level in xephyr script

### PR DESCRIPTION
**Pull request by tjarvstrand**

### CODE REVIEW

File: scripts/xephyr [hunk 1]
```diff
@@ -3,12 +3,13 @@
 HERE=$(dirname $(readlink -f $0))
 SCREEN_SIZE=${SCREEN_SIZE:-800x600}
 XDISPLAY=:1
+LOG_LEVEL=${LOG_LEVEL-INFO}
```
Comment (flacjacket) [inline scripts/xephyr]: Does this need to be `${LOG_LEVEL:-INFO}`?

Comment (tjarvstrand) [inline scripts/xephyr]: Yeah, you're right that's probably better. The way it is now works for me but I googled a bit and there is a slight difference in that ```${VAR:-VAL}``` will substitute ```VAR``` with ```VAL``` if ```VAR``` is set but null whereas ```${VAR:-VAL}``` will not. The former is probably better. You always learn something new :)

Comment (flacjacket) [inline scripts/xephyr]: Alright, I wasn't sure, which is why I asked, if you change that I'd be happy to pull it in!

**Comment by tjarvstrand** [comment]
Updated

**Comment by flacjacket** [comment]
Great, thanks!

**Pull request closed without merging**